### PR TITLE
feat(menu-item): allow icon and checkbox or radio-button (VIV-1044)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Review by YonatanKra — APPROVED** [review]
LGTM.
image

File: libs/components/src/lib/menu-item/README.md [hunk 1]
```diff
@@ -112,7 +129,7 @@ The disabled state of the element
 
 ```html preview
 <vwc-menu open>
-  <vwc-menu-item disabled text="Disabled Menu item"></vwc-menu-item>
+  <vwc-menu-item disabled text="Disabled Menu item" icon="layout-2-solid" ></vwc-menu-item>
```
Comment (YonatanKra) [inline libs/components/src/lib/menu-item/README.md:132]: Why the icon here?

Comment (rachelbt) [reply libs/components/src/lib/menu-item/README.md:133]: because we need to see also the icon color change in the disabled state

Comment (rachelbt) [reply libs/components/src/lib/menu-item/README.md:133]: come to think about it - I'll remove it and add it to the UI tests

**Review by YonatanKra — APPROVED** [review]

File: libs/components/src/lib/menu-item/README.md [hunk 2]
```diff
@@ -90,6 +90,24 @@ Note: Icon, by its own, doesn't make a discernible text. An `aria-label`, `aria-
...
+    block-size: 300px; 
+  }
+</style>
+<vwc-menu open>
+  <vwc-menu-item icon="layout-1-solid" role="menuitemcheckbox" text="Checkbox 1"></vwc-menu-item>
+  <vwc-menu-item icon="apps-solid" role="menuitemcheckbox" text="Checkbox 2"></vwc-menu-item>
+  <vwc-divider></vwc-divider>
+  <vwc-menu-item icon="layout-1-solid" role="menuitemradio" text="Radio 1.1"></vwc-menu-item>
+  <vwc-menu-item icon="apps-solid" role="menuitemradio" text="Radio 1.1"></vwc-menu-item>
+</vwc-menu open>
```
Comment (rachelbt) [inline libs/components/src/lib/menu-item/README.md:108]:
```suggestion
  <vwc-menu-item icon="image-line" role="menuitemcheckbox" text="Checkbox 1"></vwc-menu-item>
  <vwc-menu-item icon="file-1-line" role="menuitemcheckbox" text="Checkbox 2"></vwc-menu-item>
  <vwc-divider></vwc-divider>
  <vwc-menu-item icon="image-line" role="menuitemradio" text="Radio 1.1"></vwc-menu-item>
  <vwc-menu-item icon="file-1-line" role="menuitemradio" text="Radio 1.1"></vwc-menu-item>
</vwc-menu open>
```

**Review by YonatanKra — APPROVED** [review]
:)

**Pull request merged**